docs/pages/tasks/application-management.md

```diff
@@ -1,30 +1,54 @@
 ---
 layout: default
-title: Application Management
-permalink: tasks/application-management
+title: Infrastructure Management
+permalink: tasks/infrastructure-management
```

Comment:
```suggestion
permalink: tasks/infrastructure-management
redirect_from: tasks/application-management
```

@MUzairS15, we'll want to include a redirect to avoid 404s. 👍